dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/OAuth2LoginBackingBean.java

```diff
@@ -33,10 +34,11 @@
  * @author michael
  */
 @ViewScoped
+@Named("OAuth2Page")
 public class OAuth2LoginBackingBean implements Serializable {
 
     private static final Logger logger = Logger.getLogger(OAuth2LoginBackingBean.class.getName());
-    private static final long STATE_TIMEOUT = 1000 * 60 * 15; // 15 minutes in msec
+    private static final long STATE_TIMEOUT = 15 * 60 * 1000; // 15 minutes in msec
```

Comment:
Maybe ` TimeUnit.MINUTES.toMillis(15);`?

Reply:
Just changed it for aesthetics but you're right.